beta-src/src/components/ui/WDPhaseUI.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+import { Box } from "@mui/material";
+import { fetchGameOverview } from "../../state/game/game-api-slice";
+import { useAppDispatch, useAppSelector } from "../../state/hooks";
+import Season from "../../enums/Season";
+import UIState from "../../enums/UIState";
+import WDCountdownPill from "./WDCountdownPill";
+import WDGamePhaseIcon from "../svgr-components/WDGamePhaseIcon";
+import WDPillScroller from "./WDPillScroller";
+
+const WDPhaseUI: React.FC = function (): React.ReactElement {
+  const [season, setSeason] = useState<[Season, string]>([
+    Season.SPRING,
+    "1901",
+  ]);
+  const [timerDisplayState, setTimerDisplayState] = useState<boolean>(true);
+
+  const dispatch = useAppDispatch();
+  const currentSeason = useAppSelector((state) => state.game.overview.date);
```

Comment:
Thats looks way cleaner, thanks! 

Also, just so for clarification, are we able to use just gameOverview instead of stepping into the state object like I did previously because you already step into it in the game slice?
```javascript 
export const gameOverview = ({
  game: { overview },
}: RootState): GameOverviewResponse => overview;
```

Reply:
yes, if you notice gameOverview is just a function that returns the overview part of the game state...which is just a predefined export for convenience in grabbing overview state. basically doing the same thing you are doing, but with this method, we only have to call useAppSelector once to get both pieces you need.